AdblockPlusSafari/NSString+MarkdownRenderer.m

```diff
@@ -0,0 +1,48 @@
+//
+//  NSString+MarkdownRenderer.m
+//  AdblockPlusSafari
+//
+//  Created by Pavel Zdeněk on 2.O.15.
+//  Copyright © 2015 Eyeo GmbH. All rights reserved.
+//
+
+#import "NSString+MarkdownRenderer.h"
+@import UIKit;
+
+@implementation NSString (MarkdownRenderer)
+
+static NSRegularExpression* pairingRegex;
+
+- (NSAttributedString*)markdownSpanMarkerChar:(NSString*)markerChar
+                                 renderAsFont:(UIFont*)font {
+  NSError* err = nil;
```

Comment:
Pavel's implementation of markdown parser (support for localisation, bold fonts).